src/Type/WPMutationType.php

```diff
@@ -307,6 +308,7 @@ protected function register_mutation_field() : void {
 					'isPrivate'   => $this->is_private,
 					'type'        => $this->mutation_name . 'Payload',
 					'resolve'     => $this->resolve_mutation,
+					'name'        => Utils::format_field_name( $this->mutation_name ),
```

Comment:
Great catch! I'm noticing that downstream, `TypeRegistry::prepare_fields()` [just does a `lcfirst()`](https://github.com/wp-graphql/wp-graphql/blob/develop/src/Registry/TypeRegistry.php#L892). Not sure it really matters tho.